web/client/actions/styleeditor.js

```diff
@@ -186,14 +186,14 @@ export function deleteStyle(styleName) {
 * Setup the style editor service
 * @memberof actions.styleeditor
 * @param {object} service style editor service
-* @param {object} permissions editing allowed roles and groups permission object
+* @param {object} cfg configurations to be initialized for the style editor
 * @return {object} of type `INIT_STYLE_SERVICE`
 */
-export function initStyleService(service, permissions) {
+export function initStyleService(service, cfg) {
     return {
         type: INIT_STYLE_SERVICE,
         service,
-        permissions
+        cfg
```

Comment:
Tested locally and it seems to work, the only thing I would like to change is the name of the param here in the action. The `cfg` seems one to one with the localConfig.json configuration but in reality we are only managing a subset of the properties. My proposal is to change the action parameter name (eg. `options` or `config` ) to make clear this property is another object